sandbox-tests/solution-migration/language-version-upgrade/java-4/README.md

```diff
@@ -0,0 +1,34 @@
+**Using Java 8 Date Time API and types test**
+
+*Test Description*
+
+A developer is trying to rewrite code using Java 8 Date Time API and types
+
+*Act*
+
+<details>
+<summary>Code Completion instructions:</summary>
+
+- Open the project solution-migration/language-version-upgrade/java in IDE
+- Open the DateUtils class
+- Type after the method `daysBetween`:
```

Comment:
Same question here - do we give enough information to the code completion tool that we need the exact API? Maybe there could have been a comment at least like // Java 8 Date Time API. If there's a comment will the situation improve? or we may check with Cursor if it solves the case